libs/frontend/src/routes/(authenticated)/multiplayer/+page.svelte

```diff
@@ -0,0 +1,131 @@
+<script lang="ts">
+	import { authenticatedUserInfo } from "@/stores";
+	import { onMount } from "svelte";
+	import { GameEventEnum } from "types";
+
+	let state = {
+		// TODO: timeleft/countdown timer when game is started
+		creator: false,
+		gameId: undefined,
+		games: [
+			// {
+			// players
+			// amountOfPlayersJoined: number
+			// }
+		]
+	};
+
+	let socket: WebSocket;
+	onMount(() => {
+		const websocketUrl = import.meta.env.VITE_BACKEND_WEBSOCKET_MULTIPLAYER;
+
+		if (!websocketUrl) {
+			alert("Did you forget an environment variable? VITE_BACKEND_WEBSOCKET_MULTIPLAYER");
+		}
+
+		socket = new WebSocket(websocketUrl);
+
+		socket.addEventListener("open", (message) => {
+			console.log("WebSocket connection opened");
+		});
+
+		socket.addEventListener("message", (message) => {
+			const data = JSON.parse(message.data);
+			console.log({ data });
+
+			const { event } = data;
+			switch (event) {
+				case GameEventEnum.HOST_GAME:
+					{
+						state.gameId = data.message;
+					}
+					break;
+				case GameEventEnum.OVERVIEW_OF_GAMES:
+					{
+						state.games = data.message;
+					}
+					break;
+				default:
+					break;
+			}
+		});
```

Comment:
check if there are other events that need to be handled

Reply:
there definitely are! just added a new one, telling the users to go to the newly create game